dev_guide/builds.adoc

```diff
@@ -50,9 +118,9 @@ To allow access to build logs, use the following command:
 `$ oc build-logs _<buildID>_`
 ****
 
-*STI Build Logs*
+*Source Build Logs*
 
-link:../architecture/core_objects/builds.html#sti-build[STI builds] by default
+link:../architecture/core_objects/builds.html#source-build[Source builds] by default
```

Comment:
you're calling it source build here.  so the arch doc lost a change or something.